.github/workflows/verify-pull-request.yml

```
@@ -0,0 +1,38 @@
+name: Verify pull request
+
+on:
+  pull_request
+
+jobs:
+  verify_homebrew_macos_12:
+    name: MacOS 12 - Verify Homebrew formula installation
+    runs-on: macos-12
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Ockam
+        run: |
+          set -e
+
+          brew install build-trust/ockam/ockam
+          ockam --version
+
+          ls -l /usr/local/etc/bash_completion.d/ockam
+
+          echo "Ockam was successfully installed via Homebrew on MacOS 12." | tee -a $GITHUB_STEP_SUMMARY
+
+  verify_homebrew_ubuntu_22_04:
+    name: Ubuntu 22.04 - Verify Homebrew formula installation
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
```

We try to pin our dependencies, please change to: 

```suggestion
      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
```